src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -74,6 +45,17 @@ The following options can be configured as options to the integration, in `new R
 | inlineStylesheet | boolean | `true`  | Should inline stylesheets used in the recording |
 | recordCanvas     | boolean | `false` | Should record `<canvas>` elements |
 
+## Identifying Users
```

Comment:
Identifying user is an SDK specific thing. It'll improve the user experience for sure, so makes total sense to add here. I tried to word it a bit as "here are the actual docs, but here's a quick snippet to get you started"